freestyle/shared/src/main/scala/freestyle/free.scala

```diff
@@ -141,6 +141,7 @@ object freeImpl {
 
           import _root_.freestyle.{ FreeS, FSHandler, InjK }
 
+          type OpTypes = _root_.iota.KCons[Op, _root_.iota.KNil]
```

Comment:
Why do we need to change the `@free` macro? The `Op` is a constant name, not a fresh type name.